Automattic/wp-calypso
# Tracks: Added theme_type property when activating a theme

Related to https://github.com/Automattic/wp-calypso/issues/82701

## Proposed Changes

* Added the theme_type property to the calypso_theme_sheet_primary_button_click event.

## Testing Instructions



* Open the Calypso Live Link and navigate to: /theme/smithland/{site}?tab_filter=recommended
* Open your browser's network tab and filter by `calypso_theme_sheet_primary_button_click`
* Click on the button that activates the theme.
* You should see the `calypso_theme_sheet_primary_button_click` event in the network tab.
* Inspect the event and you should see that the theme_type exists and has the correct value.
* Try with multiple theme types.

## Pre-merge Checklist
